### PR TITLE
Add twitter (share) button

### LINKED ISSUE #72: Add sharing options to blog posts
Looks like we want to have sharing options on each blog post:

- Twitter
- Pocket

For QA: current mocks

Screen Shot 2021-04-19 at 15 42 35
Screen Shot 2021-04-19 at 15 42 30

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #72

---

UTM params might change.

## Screenshots

Screen Shot 2021-04-19 at 12 30 07

Screen Shot 2021-04-20 at 15 54 01

### CODE REVIEW

File: src/assets/css/blogposts.scss [hunk 1]
```diff
@@ -151,7 +151,36 @@ $pull-quote-color: #4e1a6a;
...
+    border-radius: 0;
+    border-style: solid;
+    border-width: 2px;
+    color: $white;
+    display: inline-block;
+    padding: 2px 10px 2px 35px;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    transition: background-color 100ms ease-in-out;
```
Comment (willdurand) [inline src/assets/css/blogposts.scss:171]: Thanks blog.mozilla.org :)

**Comment by willdurand** [comment]
mmm I just noticed that the mocks have a different layout on small screens:

Screen Shot 2021-04-19 at 15 41 31

**Comment by willdurand** [comment]
> mmm I just noticed that the mocks have a different layout on small screens:
> 
> Screen Shot 2021-04-19 at 15 41 31

fixed.

**Comment by willdurand** [comment]
@muffinresearch it's now ready to review

File: src/assets/css/blogposts.scss [hunk 2]
```diff
@@ -133,25 +134,61 @@ $pull-quote-color: #4e1a6a;
...
+
+    &:hover,
+    &:focus {
```
Comment (muffinresearch) [inline src/assets/css/blogposts.scss:180]: Does this need something for `&:active`?

**Review by muffinresearch — APPROVED** [review]
r+wc

**Pull request merged**